app.js

```diff
@@ -36,6 +39,12 @@ app.use((err, req, res, next) => {
         case '22P02':
             res.status(400).send({ msg: 'Invalid input' })
             break
+        case '23502':
+            res.status(400).send({ msg: 'Invalid input (not_null_violation)' })
+            break
+        case '23503':
+            res.status(404).send({ msg: 'Invalid input (foreign_key_violation)' })
+            break
```

Comment:
I can see you have an if statement above, I'd recommend keeping it consistent across your code.😄